server/graphql/common/expenses.ts

```diff
@@ -1781,13 +1815,12 @@ export async function payExpense(req: express.Request, args: Record<string, unkn
     const useTwoFactorAuthentication =
       isTwoFactorAuthenticationRequiredForPayoutMethod && !forceManual && hostHasPayoutTwoFactorAuthenticationEnabled;
 
+    const expensePaidAmountKey = `${req.remoteUser.id}_2fa_payment_limit`;
+    let currentPaidExpenseAmount;
```

Comment:
:ice_cream: I find these variable names a bit misleading, I initially thought it was about the amount of the current expense. Suggestions:
- `alreadyPaidExpensesAmountKey`, `totalPaidExpensesAmountKey`
- `alreadyPaidExpensesAmount`, `totalPaidExpensesAmount`